src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -1,23 +1,24 @@
 package core.basesyntax;
 
-/**
- * <p>Реалізуйте метод `getUniqueCharacters(String fileName)` який приймає як параметр назву файлу.
- * Для цього використовуйте TreeSet. Файл містить букви латинського алфавіту і розділові знаки.</p>
- *
- * <p>Метод повинен відсортувати всі букви по алфавіту і повернути строку з 5 різних букв без
- * розділових знаків.
- * Якщо у файлі міститься менше п'яти різних букв, то потрібно повернути їх усі.
- * Можливий випадок, коли переданого файлу не існує, в такому разі потрібно викинути помилку
- * про відсутність файлу.</p>
- *
- * <p>Приклад 1: ur-BvT?^ ra w; p
- * Результат 1: abprt</p>
- *
- * <p>Приклад 2: A _f*c a?F
- * Результат 2: acf</p>
- */
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.TreeSet;
+
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        TreeSet<Character> treeSet = new TreeSet<>();
+        try (FileReader fileReader = new FileReader(fileName)) {
+            boolean hasNext = fileReader.ready();
+            while (hasNext) {
+                char a = (char) fileReader.read();
+                if (Character.isAlphabetic(a)) {
```

Comment:
```suggestion
                if (Character.isLetter(a)) {
```
тоді в подальшому не доведеться робити replaceAll